app/uk/gov/hmrc/nationalinsurancerecord/domain/NationalInsuranceRecord.scala

```diff
@@ -33,6 +35,8 @@ case class NationalInsuranceRecord(
                                   )
 
 object NationalInsuranceRecord {
+  implicit val jodaWrites: Writes[LocalDate] = jodaLocalDateWrites("yyyy-MM-dd")
```

Comment:
suggestion

I think there are some implicits in the joda library if you ever need them in the future

import play.api.libs.json.JodaWrites._
import play.api.libs.json.JodaReads._

Reply:
Good spot